homeassistant/components/filter/sensor.py

```diff
@@ -191,6 +192,7 @@ def __init__(self, name, entity_id, filters):
         self._filters = filters
         self._icon = None
         self._device_class = None
+        self._attr_state_class = None
```

Comment:
This is the default, thus duplicate

```suggestion
```

Reply:
`AttributeError: 'SensorFilter' object has no attribute '_attr_state_class'`

Reply:
It's not set in the base class, so it's correct to initialize it to None here.

Reply:
Oh right! My bad!